README.md

```diff
@@ -358,7 +360,7 @@ spec:
       value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/ui:latest
 ```
 
-And the following `PipelineResource` defines the git repository for the backend application:
+The git repository for the backend application:
```

Comment:
why did you remove the info about PipelineResource?

Reply:
Because every where PipelineResource was repeating so just added `PipelineResource` in the first place followed by explanation